Documentation/concepts/kubernetes/compatibility-table.rst

```diff
@@ -98,15 +100,9 @@
 +-----------------+----------------+
 | v1.8.5          | 1.21.2         |
 +-----------------+----------------+
-| v1.8            | 1.21.2         |
-+-----------------+----------------+
-| v1.9.0-rc0      | 1.22.1         |
+| v1.8.6          | 1.21.2         |
 +-----------------+----------------+
-| v1.9.0-rc1      | 1.22.2         |
-+-----------------+----------------+
-| v1.9.0-rc2      | 1.22.2         |
```

Comment:
Why are those rc lines removed for v1.9.0 but not for v1.8.0?

Reply:
because this is 1.8 documentation, they shouldn't have been added in the first place.

Reply:
Ah, of course :facepalm: